app/src/androidTest/java/org/mozilla/tv/firefox/integration/LiveDataMapTest.kt

```diff
@@ -18,8 +18,7 @@ import org.mozilla.tv.firefox.helpers.MainActivityTestRule
 // the linting error for starting a function with an uppercase letter
 class LiveDataMapTest {
 
-    @Rule @JvmField
-    val activityTestRule = MainActivityTestRule()
+    @Rule @JvmField val activityTestRule = MainActivityTestRule()
```

Comment:
I like the convention of inlining annotations, but it should be mentioned that it violates JetBrains and Android style guides.  We should probably decide on one strategy with regards to this and apply it everywhere.

https://kotlinlang.org/docs/reference/coding-conventions.html#annotation-formatting
https://developer.android.com/kotlin/style-guide#annotations

It is however supported by the Square Android style guide (I can't find a link in natural language, it's in here somewhere https://github.com/square/java-code-styles/blob/master/configs/codestyles/SquareAndroid.xml).

Reply:
🦅 eyes over here! I remember seeing `@Whatever fun...` is okay so I assumed two annotations would also be okay. Guess not! I'm fine placing the annotations together, above the fun. That being said, `@get:Rule` invalidates this.

Reply:
Hah, we used the Square rules at my last job.  

Both guides claim that inlining a single annotation with no arguments is fine.  I'm not sure if `@get:Rule` counts as one or two, but it could potentially be allowed.